README.md

```diff
@@ -32,6 +33,36 @@ Set the path to `libcrypto.a` either as a command line argument to gradle `-Plib
 
 ### OSX
 
+#### Debugging simulator test issues
+
+**Xcode does not support simulator tests for <native target>**
```

Comment:
**Nit**: `<native target>` does not render because GH-flavored markdown interprets `<` as the start of a tag. Escape as `\<native target>`.